.storybook/blocks/Hint.jsx

```diff
@@ -1,16 +1,11 @@
+import { compiler } from "markdown-to-jsx"
 import styles from "./hint.module.css"
 
-/**
- * A block that displays the story name or title from the of prop
- * - if a story reference is passed, it renders the story name
- * - if a meta reference is passed, it renders the stories' title
- * - if nothing is passed, it defaults to the primary story
- */
```

Comment:
I removed this because the description was incorrect. We can add jsdoc comments back later.